python/lsst/pipe/base/tests/simpleQGraph.py

```diff
@@ -500,7 +500,7 @@ def makeSimpleQGraph(
         pipeline_graph,
         butler,
         skip_existing_in=skipExistingIn if skipExistingIn is not None else [],
-        input_collections=butler.collections if butler.collections is not None else [run],
+        input_collections=butler.collections.defaults if butler.collections.defaults is not None else [run],
```

Comment:
I am wondering why we had the `is not None` there. Wouldn't `if butler.collections.defaults else` work to cover None and empty list? Does the new way of doing it ever return `None`?

Reply:
Must be deep historical reasons -- even prior to your replacement of `butler.collections` I don't think this could have been `None`.  Both versions just look at `RegistryDefaults.collections` which is an empty tuple if there are no collections.

Reply:
The test code presumably never ended up with `[run]` then.